src/erl.rs

```diff
@@ -137,6 +143,22 @@ pub fn record_definition(name: &str, fields: &[&str]) -> String {
     buffer
 }
 
+
+pub fn js_module(module: &TypedModule) -> String {
```

Comment:
Could we put these in a different module? `javascript.rs` perhaps?

Reply:
definitely, I'm really just poking around now. I though I might be able to use some of the helpers like formatting args, but I actually think that is of minimal benefit now